### ISSUE TITLE
Add support to systemd timers

### ISSUE THREAD

**Issue by tonyskapunk** [rackerlabs/kthresher#48]
References to timers:
  - http://0pointer.de/public/systemd-man/systemd.timer.html
  - https://www.freedesktop.org/software/systemd/man/systemd.timer.html

Examples:
  - https://wiki.archlinux.org/index.php/Systemd/Timers#Examples

**Comment by komish** [comment]
I'll take a stab at this. If possible, please assign it my way.

**Comment by tonyskapunk** [comment]
Thanks! if it helps take a look to these:
- https://github.com/rackerlabs/recap/blob/master/src/utils/recap.timer
- https://github.com/rackerlabs/recap/blob/master/src/utils/recap.service.in

**Comment by tonyskapunk** [comment]
Can't assign if you are not part of the team but I can create a tag for you ;)

**Comment by komish** [comment]
@tonyskapunk I think github did not close this issue because a second PR references this issue as well and that PR is still open. Just a heads up.

**Comment by tonyskapunk** [comment]
That's correct. This should be closed once the new PR is merged, thanks!

**Comment by tonyskapunk** [comment]
Closing this one now! Thanks for the help @Komish
